tests/foreman/maintain/test_backup_restore.py

```diff
@@ -35,9 +35,8 @@
 BASIC_FILES = {"config_files.tar.gz", ".config.snar", "metadata.yml"}
 CONTENT_FILES = {"pulp_data.tar", ".pulp.snar"}
 OFFLINE_FILES = {"pgsql_data.tar.gz", ".postgres.snar"}
-ONLINE_SAT_FILES = {"candlepin.dump", "foreman.dump", "pulpcore.dump", "pg_globals.dump"}
+ONLINE_SAT_FILES = {"candlepin.dump", "foreman.dump", "pulpcore.dump"}
 ONLINE_CAPS_FILES = {"pulpcore.dump"}
-REMOTE_SAT_FILES = ONLINE_SAT_FILES - {"pg_globals.dump"}
```

Comment:
I think this @vsedmik added this earlier for remotedb testing, so are you not going to test backup restore against remotedb?

Reply:
I think Griffin is fixing the fact that `pg_globals.dump` was removed from 6.13 backups, so now `ONLINE_SAT_FILES == REMOTE_SAT_FILES`, if I got it right.

Reply:
yep @vsedmik is correct. We no longer use `pg_globals.dump`. See https://bugzilla.redhat.com/show_bug.cgi?id=2172540